### ISSUE TITLE
Scroll-to-top button disturbs left-handed users experience

### ISSUE THREAD

**Issue by lemoscaio** [bluesky-social/social-app#5619]
### Steps to Reproduce

1 - Use the left hand to scroll down the timeline

You'll notice that sometimes you'll press the button by mistake, or you will trigger a horizontal scroll accidentally.

### Attachments

https://github.com/user-attachments/assets/c751ba99-77d9-459b-a5b6-44dd451ef207

### What platform(s) does this occur on?

Android

### Device Info

Samsung Galaxy S20+

### What version of the app are you using?

Build version: 1.91.2.324; Bundle info: b2b5be5 (prod); Bundle date: 24091719; Platform: android

### Additional Information

Right-handed users may not understand how this is bad because how we're conditioned to use depends on our dominant hand. Left-handed users are used to have the left area free for scrolling

An example of how the dominant hand can change the way we use the device is how left-handed users press the volume buttons of a smartphone (usually with the index finger instead of the thumb):

![image](https://github.com/user-attachments/assets/26200256-d80e-4cd6-86fb-9c98fe0c7e03)

**Comment by SpriterGors** [comment]
The same happens to a smaller degree with me, a right-handed user, when I accidentally slide my thumb over the post composer button.

Perhaps an option to place them on the top half of the device screen could be useful?

**Comment by Jerrynicki** [comment]
it'd be awesome if it were possible to press-and-hold the button to move it to an arbitrary location. an example of this is the "skip to next comment" button that's shown on the bottom-right in the official Reddit app when inside a post's comments

**Comment by lemoscaio** [comment]
To be honest, I think that only showing the scroll-to-top button when the user is scrolling up is enough thinking about the complexity of other changes. It's the common behavior I see in other apps and it works perfectly (at least I was never bothered by this). 

The header with the tabs already has this behavior of showing up again when the user scrolls up, so it would probably be just a new condition to show the scroll-to-top button. I tried to find it in the code to make a better suggestion, but I'm not so familiar with React Native and wasn't able to debug what makes the header appear.

**Comment by gaearon** [comment]
I think we should just nix that button and move the dot into the tab bar which is pretty standard. Second tap on the bar already scrolls you up.

**Comment by lemoscaio** [comment]
> I think we should just nix that button and move the dot into the tab bar which is pretty standard. Second tap on the bar already scrolls you up.

Makes sense. I never noticed this behavior in some apps, just did it naturally 😅.
